website/docs/components/counter/counter-a11y.md

```diff
@@ -4,14 +4,14 @@ a11y: AA
 tabs: Design('counter'), A11y('counter-a11y'), API('counter-api'), Example('counter-code'), Changelog('counter-changelog')
 ---
 
-Counter component doesn't require any special accessibility considerations, but make sure to include it in the reading order so screen readers can access it. Since it's not interactive, it doesn't need to be in the focus (tab) order.
-
-Avoid the use of aria-labels on `span` elements.
-
-## Considerations for designers
+## Considerations for designers and developers
 
 - Counter text should be clear and concise.
 - Make sure to [check the contrast of the counter text against the background](/core-principles/a11y/a11y-design#color_and_contrast).
```

Comment:
I would rewrite this point slightly differently. 

**Reason:** the contrast of the text within the components is managed on the design system side, and designers don't need to additionally check the text within the counters. 

**We could write something like:** If you are using a counter with the default theme on a colored or dark background, please check the contrast between the text and the background. Something like that.

Reply:
updated the text